crypto/dso/dso_win32.c

```diff
@@ -12,8 +12,8 @@
 
 #if defined(DSO_WIN32)
 
-# ifdef _WIN32_WCE
-#  if _WIN32_WCE < 300
+# if defined(_WIN32_WCE) || defined(OPENSSL_SYS_WIN_CORE)
+#  if _WIN32_WCE < 300 && !defined(OPENSSL_SYS_WIN_CORE)
```

Comment:
I don't understand why is this needed. onecore.lib has GetProcessAddress and LoadLibraryA. In other words it should work without any modifications.

Reply:
That change was actually aiming for mincore.lib used by Windows IoT
mincore is a subset of oneCore, which is unfortunetely not supporting legacy LoadLibrary method
I Agree it's not needed to be OneCore capable, its just enough to add ExLibraries as OneCore (or temporarly OneCore and its downlevel version), and also disable few methods not supported by core API, and make sure no WinAPI is called directly.